lms/templates/courseware/course_navigation.html

```diff
@@ -78,6 +78,7 @@
              </a>
           </li>
       % endfor
+      <%block name="extratabs" />
```

Comment:
This line accidentally got lost when removing the Staff/Student toggle. I'm thinking once it has merged I'll squash this commit into the preview commit so that the bug is never introduced.